### ISSUE TITLE
can you share a normal training log?

### ISSUE THREAD

**Issue by poodarchu** [RenzKa/simlingo#19]
I have started the trianing with vlm, but the console show only progress bar of some tasks looks like "filtering data", but no training loss is displayed, nor did I found loss related logs in wandb files.

**Comment by poodarchu** [comment]
[STDOUT] 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 15991/15991 [2:22:22<00:00,  1.87it/s]
[STDOUT] 
[STDOUT] [train samples]: Loading 84014 images from /database/simlingo_v2_2025_01_10 for bucket vehicle_side
[STDOUT] 
[STDOUT] Total amount of routes:
[STDOUT] 
[STDOUT] 31982
[STDOUT] 
[STDOUT] Crashed routes:
[STDOUT] 
[STDOUT] 1187
[STDOUT] 
[STDOUT] Perfect routes:
[STDOUT] 
[STDOUT] 14804
[STDOUT] 
[STDOUT] Fail reasons:
[STDOUT] 
[STDOUT] {'measurement_file_not_in_bucket': 966842, 'measurement_folder_not_in_bucket': 972927, 'route_crashed': 1176, 'no_results.json': 11}
[STDOUT] 
[STDOUT] Found 16153 routes in /mnt/volumes/base-3da-ali-sh/bjzhu/repos/simlingo/database/simlingo_v2_2025_01_10
[STDOUT] 
[STDOUT] Use 15991 routes.
[STDOUT] 
[STDOUT] 0%|                                                                                                                                                                                      | 0/15991 [00:00<?, ?it/s]
[STDOUT] 0%|                                                                                                                                                                              | 1/15991 [00:00<44:44,  5.96it/s]
[STDOUT] 0%|                                                                                                                                                                              | 2/15991 [00:00<47:00,  5.67it/s]
[STDOUT] 0%|                                                                                                                                                                              | 3/15991 [00:00<45:28,  5.86it/s]
[STDOUT] 0%|                                                                                                                                                                              | 4/15991 [00:00<49:48,  5.35it/s]
[STDOUT] 0%|                                                                                                                                                                              | 5/15991 [00:00<49:56,  5.33it/s]
[STDOUT] 0%|                                                                                                                                                                            | 6/15991 [00:01<1:03:31,  4.19it/s]
[STDOUT] 0%|                                                                                                                                                                            | 7/15991 [00:01<1:00:26,  4.41it/s]
[STDOUT] 0%|                                                                                                                                                                              | 8/15991 [00:01<56:04,  4.75it/s]
[STDOUT] 0%|                                                                                                                                                                              | 9/15991 [00:01<56:53,  4.68it/s]
[STDOUT] 0%|                                                                                                                                                                             | 10/15991 [00:02<57:37,

it seems loop infinitely to iterate the data, with no training loss printed.

the above log is extract from the log.wandb file, basically the same as in the terminal.

**Comment by poodarchu** [comment]
how many buckets are there in total?

**Comment by RenzKa** [comment]
You can see the buckets here: https://github.com/RenzKa/simlingo/blob/main/simlingo_training/config/data_module/carla_bucket_v12_dreamer.yaml

I would recommend to test the training without using the buckets and on a smaller dataset, to see if training starts. For that you can change the bucket setting in your config file, e.g. change `carla_bucket_v12_dreamer` to `carla_no_buckets` here: https://github.com/RenzKa/simlingo/blob/edaa8e7a91a12ebec1a1c556b3ead5857c211e7d/simlingo_training/config/experiment/simlingo_seed1.yaml#L3 

And add this line back in, and maybe reduce to a even lower number to speed up data loading:
https://github.com/RenzKa/simlingo/blob/edaa8e7a91a12ebec1a1c556b3ead5857c211e7d/simlingo_training/dataloader/dataset_base.py#L218

**Comment by poodarchu** [comment]
thanks a lot~
